### PR TITLE
Adjust invate user accordingly to use-case

### PR DESCRIPTION
**Pull request by ANGIECATHERINEP**
Adjust invate user accordingly to use-case

### CODE REVIEW

**Review by LinconOssa — APPROVED** [review]
It is approved with something missing, which is the date when one believes that we do not know how to solve it so that it is not like this:
image

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/components/feedback/InteractiveModal/types.ts [hunk 1]
```diff
@@ -17,7 +17,8 @@ interface InteractiveModalProps {
   infoData: { [key: string]: string } | Record<string, string | number>;
   searchData?:
     | Record<string, string | number | unknown>[]
-    | { [key: string]: string };
+    | { [key: string]: string }
+    | any;
```
[the remark this answers is not in the document]
Comment (ANGIECATHERINEP) [reply src/components/feedback/InteractiveModal/types.ts]: resolved

File: src/components/feedback/InteractiveModal/types.ts [hunk 2]
```diff
@@ -30,8 +31,8 @@ interface InteractiveModalProps {
   type?: ModalTypes;
   divider?: boolean;
   onClick?: any;
-  idLabel?: string;
-  nameLabel?: string;
+  idLabel: string;
```
[the remark this answers is not in the document]
Comment (ANGIECATHERINEP) [reply src/components/feedback/InteractiveModal/types.ts]: resolved

File: src/pages/privileges/outlets/users/invite/index.tsx [hunk 3]
```diff
@@ -9,21 +9,22 @@ import { validationMessages } from "@validations/validationMessages";
 
 import { InviteUI } from "./interface";
 import { IInviteFormValues } from "./types";
-import { userEntriesDataMock } from "@src/mocks/apps/privileges/users/users.mock";
 import { userSearchCardData } from "@src/mocks/apps/privileges/users/usersSearchField.mock";
+import { saveLinixInvitations } from "./utils";
+import { invitationUserEntriesDataMock } from "@src/mocks/apps/privileges/invitations/invitationUsers.mock";
```
[the remark this answers is not in the document]
Comment (ANGIECATHERINEP) [reply src/pages/privileges/outlets/users/invite/index.tsx]: resolved

**Pull request merged**